### ISSUE TITLE
[Update] Instructions for migrating data from existing PG table

### ISSUE THREAD

**Issue by mfreed** [timescale/docs.timescale.com-content#627]
Our docs provides isntructions for migrating from the same database, but appears to have been written prior to the support for `migrate_data` in a hypertable.   Specifically, it suggests creating a second table, then running `INSERT INTO new_table SELECT * FROM old_table;`

https://docs.timescale.com/latest/getting-started/migrating-data#same-db

We should update these instructions to instead just point at `migrate_data` feature/parameter of hypertables:

https://docs.timescale.com/latest/api#create_hypertable

**Comment by rept** [comment]
Definitely!  Didn't know this was possible.  Since this saves people from having to over dimension their machines in order to migrate.  Is it also possible to immediately convert an existing table to a hypertable with compression as well?
